nikola/conf.py.in

```diff
@@ -343,6 +356,25 @@ HIDDEN_TAGS = ['mathjax']
 # However, more obscure tags can be hidden from the tag index page.
 # TAGLIST_MINIMUM_POSTS = 1
 
+# A list of dictionaries specifying tags which translate to each other.
+# For example:
+#   [
+#     {'en': 'car', 'de': 'Auto'},
+#     {'en': 'window', 'fr': 'fenêtre'},
+#   ]
+# Note that tags in the same language can appear more than once, for
+# example if they have different meaning:
+#   [
+#     {'en': 'cloud', 'de': 'Wolke'},  # the fluffy thing in the sky
+#     {'en': 'cloud', 'de': 'Cloud'},  # cloud computing
+#   ]
+# TAG_TRANSLATIONS = []
+
+# If set to True, a tag in a language will be treated as a translation
+# of the literally same tag in all other languages. Enable this if you
+# do not translate tags, for example.
+# TAG_TRANSLATIONS_ADD_DEFAULTS = False
```

Comment:
I thought that the value in the comment should be the default value, so that if you uncomment it nothing will change (at least in this version). Or is that not the case?